lib/shared/bucketing-assembly-script/assembly/helpers/jsonHelpers.ts

```diff
@@ -46,6 +46,27 @@ export function isValidString(jsonObj: JSON.Obj, key: string, validStrings: stri
     return value
 }
 
+export function isUsableString(jsonObj: JSON.Obj, key: string, validStrings: string[]): string {
+    const str = getStringFromJSON(jsonObj, key)
+    if (!validStrings.includes(str)) {
+        console.log(`[DevCycle] Warning: String value: ${str}, for key: ${key} does not match a valid string.`)
```

Comment:
should this be using `console.warn`?

Reply:
I wasn't able to find any examples that actually use the other log levels for WebAssembly, that's why I left it as a `console.log`, but let me check again

Reply:
updated to `console.warn`, it'll now look lik:
![image](https://user-images.githubusercontent.com/91074031/181105144-a8509fa5-f364-4344-8b00-fe5026d9acb9.png)